_overviews/scala3-book/ca-context-bounds.md

```diff
@@ -22,29 +23,54 @@ In that case you don’t have to define a parameter name, and can just provide t
 
 For example, this `maximum` method takes a _context parameter_ of type `Ord`, only to pass it on as an argument to `max`:
 
+{% tabs context-bounds-max-named-param %}
+
+{% tab 'Scala 3 Only' %}
+
 ```scala
 def maximum[A](xs: List[A])(using ord: Ord[A]): A =
   xs.reduceLeft(max(ord))
 ```
 
+{% endtab %}
+
+{% endtabs %}
+
 In that code the parameter name `ord` isn’t actually required; it can be passed on as an inferred argument to `max`, so you just state that `maximum` uses the type `Ord[A]` without giving it a name:
 
+{% tabs context-bounds-no-param-name %}
+
+{% tab 'Scala 3 Only' %}
```

Comment:
ok so anonymous context parameter truly is a scala 3 only thing, so if you add Scala 2 tab to `context-bounds-max-named-param`, then this paragraph and snippet can go in the scala 3 tab there (i.e. drop the `context-bounds-no-param-name` tabs).

Reply:
Actually, maybe it’s better to have it in this file. If you think it should be here, let me know and I’ll update it.